src/components/MessagingWindow/index.jsx

```diff
@@ -1,40 +1,30 @@
 import React, { Component } from "react";
-import "./MessagingWindow.css";
-import Firebase from "firebase";
 import PropTypes from "prop-types";
+import "./MessagingWindow.css";
+import firebaseWrapper from "../../firebaseWrapper";
 import Message from "./Message";
 
 class MessagingWindow extends Component {
   constructor(props) {
     super(props);
 
-    const { user, receiver } = this.props;
+    const { conversationId, user } = this.props;
     this.state = {
       messages: [],
       user,

```

Comment:
wait actually why do you store user and conversationId in state? you can just access them through props. Also rename to userId